packages/components/dialog/src/dialog.ts

```diff
@@ -65,6 +66,8 @@ export class Dialog extends ScopedElementsMixin(LitElement) {
   /** @internal */
   static override styles: CSSResultGroup = [breakpoints, styles];
 
+  #focusTrap?: FocusTrap;
```

Comment:
How about making this a `FocusTrapController`? And hide the logic inside the controller?

Reply:
That way we also don't use `focus-trap` directly *except* inside the controller?